aws-ssm-document/src/main/java/com/amazonaws/ssm/document/DocumentModelTranslator.java

```diff
@@ -117,6 +118,13 @@ UpdateDocumentRequest generateUpdateDocumentRequest(@NonNull final ResourceModel
                 .build();
     }
 
+    UpdateDocumentDefaultVersionRequest generateUpdateDocumentDefaultVersionRequest(@NonNull final String name, @NonNull final String latestVersion) {
```

Comment:
nit: the method name is general but the parameter name is specified as latestVersion.